apps/frontend/src/assets/css/variables.css

```diff
@@ -0,0 +1,25 @@
+:root {
+	/* Background Colors */
+	--background-primary: #121415;
+	--background-secondary: #1d1e21;
+	--background-hover: rgba(255 255 255 / 6%);
```

Comment:
There is an ability to add alpha values to hex color, let's use it to keep everything the same style